adobe/react-spectrum
# fix: [BUG] - Autocomplete - Text selection with Shift + Home/End isn't working (#7228)

Closes #7228

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

1. Open RAC ComboBox component example
2. Enter a word in the input, like "Flamingo"
3. Press ArrowDown to expand the ListBox
4. Press ArrowLeft to move the cursor within the input to somewhere in the middle of the entry
5. Press Shift+Home  (fn+Shift+ArrowLeft on a laptop) to select from the cursor to the beginning of the entry.
6. Press Shift+End  (fn+Shift+ArrowRight on a laptop) to select from the cursor to the end of the entry.
7. Use ArrowDown to move focus to an item in the ListBox and to navigate to an item in the middle of the list.
8. Press Home  (fn+ArrowLeft on a laptop) to move focus to the first item in the list.
9. Press End  (fn+ArrowRight on a laptop) to move focus to the last item in the list.
10. Repeat from step 4.

When there is a focusedKey in the list, Home or End will navigate within the list, but with no focusedKey, Shift+Home/End should select within the input.

## 🧢 Your Project:

Adobe/Accessibility
